pkg/markdown/markdown.go

```diff
@@ -4,18 +4,18 @@ import (
 	"os"
 	"strings"
 
-	"github.com/charmbracelet/glamour"
+	"github.com/cli/cli/internal/glamour"
 )
 
-func Render(text, style string, baseURL string) (string, error) {
```

Comment:
For the sake of minimal diff, I would suggest that this method stays the same as before.

I don't see value in splitting it into several `RenderWith*` methods, especially since that required changing all the current call sites, but there was no gain in functionality. We pass down all the same arguments (style and base URL) like before, and we hardcode the wrap margin to 80 like before.

In the future we can expose setting wrap margin to something else other than 80 (ideally the width of the user's terminal), but let's do that in a separate PR rather than in the one that is about emoji.